orc8r/cloud/go/tools/swaggergen/generate/generate.go

```diff
@@ -112,43 +110,30 @@ func GenerateModels(targetFilepath string, configFilepath string, rootDir string
 		return errors.Wrapf(err, "target filepath %s is invalid", targetFilepath)
 	}
 
-	defer func() {
-		// Always perform cleanup step
-		r := recover()
-		_ = os.RemoveAll(tmpGenDir)
-		if r != nil {
-			panic(r)
-		}
-	}()
+	tmpGenDir, err := ioutil.TempDir("./", "tmpgen")
```

Comment:
Does this work
```suggestion
	tmpGenDir, err := ioutil.TempDir(".", "tmpgen")
```

Reply:
Yes. `"."` works and it's better. Changed. Thanks!